glooshot/docs/installation/install.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Quick Start
+menuTitle: Quick Start
+weight: 2
+---
+
+## Quick start
+
+- Download the latest release of glooshot
+- Copy it to your path as an executable
+- Configure `kubectl` to point to your cluster
+
+```bash
+glooshot init
+```
```

Comment:
I think you should spend a few words describing "what" gets installed ... like that SuperGloo is a dependency, and a lot of prometheus infra gets stood up... I'd think users would be interested in seeing what those are here in the docs and why they're there.

Reply:
Also: I see that components get installed into `glooshot` namespace which is not consistent with the others (ie, `gloo-system` and `supergloo-system` curious why that is?

Reply:
good point, added some notes
I don't have a strong opinion about the namespace, probably would have chosen to use `glooshot-system` if I had thought of gloo and supergloo at the time